_includes/footer.html

```diff
@@ -46,7 +46,7 @@ <h2><a {% if column.new_window %}target="_blank" {% endif %}
 </footer>
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 {% if jekyll.environment == 'production' %}
-<script src="{{ site.baseurl }}/js/stellarnav.min.js"></script>
```

Comment:
i don't think this change is correct. in production environment we should use .min.js file because they are minified version of js file

Reply:
stellarnav.min.js is auto generated from stellarnave.js.